OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -320,8 +320,12 @@ protected function configureOptions(OptionsResolverInterface $resolver)
             'auth_with_one_url'   => false,
         ));
 
-        $resolver->setAllowedValues(array(
-            'csrf' => array(true, false),
-        ));
+        if (version_compare(Kernel::VERSION, '2.6', '>=')) {
```

Comment:
Detect one of the other changes done at the same time in OptionsResolver (``method_exists($resolved, 'setDefined')`` for instance). the other option is to use ReflectionMethod to know the number of parameters, but this will be slower